### ISSUE TITLE
[Feature]: Show mastery points after skill level reaches 10

### ISSUE THREAD

**Issue by PeckinPaw** [Annosz/UIInfoSuite2#479]
### Description

In order to more easily track progress for the Mastery Cave, please consider showing current points level as a graph when the player is doing things like fishing, foraging, mining, etc. This would be similar to showing the graph for specific skills, but after those skills are all at 10 it would be helpful to see at a glance how the Mastery Cave points are progressing.

**Comment by NoahGibby** [comment]
Yes, this feature would be great. Along with this, adding an option to toggle exp numbers to still show until you have fully maxed out the mastery cave or just always have them on would be nice too. One of my favorite features of this mod is seeing exp for everything I do while leveling skills to 10. It's just really fun to learn the amount of exp each task grants and get the instant feedback of progress. Thanks!

**Comment by djuniah** [comment]
Would just like to voice my support for both of the features mentioned in this thread as well.  I may look into submitting a pull for it if the maintainer is open to contributions (at least for the exp pop-ups).
